Formula/c/catboost-cli.rb

```diff
@@ -0,0 +1,144 @@
+class CatboostCli < Formula
+  desc "Fast, scalable, high performance Gradient Boosting on Decision Trees cli tool"
+  homepage "https://catboost.ai"
+  url "https://github.com/catboost/catboost/archive/refs/tags/v1.2.1.tar.gz"
+  sha256 "b1cf4458415a639f26e953bc4e2877d68d2a711f457a725ef93cbe24363725f0"
+  license "Apache-2.0"
+  head "https://github.com/catboost/catboost.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "conan@1" => :build
+  depends_on "ninja" => :build
+
+  uses_from_macos "llvm" => :build
+
+  def install
+    extra_cmake_args = [
+      "-S",
+      buildpath.to_s,
+      "-B",
+      "#{buildpath}/brew-build",
+      "-G",
+      "Ninja",
+      "-DCATBOOST_COMPONENTS=app",
+      "-DHAVE_CUDA=NO",
+      "-DCMAKE_TOOLCHAIN_FILE=#{buildpath}/build/toolchains/clang.toolchain",
+    ]
+    system "cmake", *extra_cmake_args, *std_cmake_args
+    system "ninja", "-C", "#{buildpath}/brew-build", "catboost"
```

Comment:
This is cleaner than adding a ninja dependency. What is the use of `CMAKE_TOOLCHAIN_FILE`?
```suggestion
    args = [
      "-DCATBOOST_COMPONENTS=app",
      "-DHAVE_CUDA=NO",
      "-DCMAKE_TOOLCHAIN_FILE=#{buildpath}/build/toolchains/clang.toolchain",
    ]
    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
Done!

Although `system "cmake", "--install", "build"` does nothing. So I've replaced this line by:

```ruby
bin.install apppath/"catboost"
```

Reply:
I've reverted this change, and now the formula is compiling right using ninja...